pact-broker/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.service }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.appName }}
+  labels:
+    app: {{ .Values.appName }}
+spec:
+  ports:
+    - name: {{ .Values.service.port | default "8080" }}-tcp
+      port: {{ .Values.service.port | default "8080" }}
+      protocol: TCP
+      targetPort: {{ .Values.service.port | default "8080" }}
+  selector:
+    app: {{ .Values.appName }}
```

Comment:
this end up pointing to the database pod ? i had to change to:

  selector:
    deploymentconfig: {{ .Values.appName }}